static/.eslintrc.js

```diff
@@ -0,0 +1,255 @@
+/* @file eslintrc
+ * @brief ESLintRC for ScottyLabs JS code.
+ *
+ * This is a pretty freaking (or flippin', as our Tech Director would say)
+ * pedantic set of rules.  100% compliance is in no way required, but it helps
+ * find some egregious stuff (and also helps with some subtle bugs that might be
+ * otherwise hard to catch.
+ *
+ * @author Oscar Bezi (bezi@scottylabs)
+ */
+
+'use strict';
+
+module.exports = {
+  'ecmaFeatures': {
+    'arrowFunctions': true,
+    'jsx': true,
```

Comment:
(Not sure where else to put this, because I'm commenting on filenames) I've never seen a `*.react.js` extension. React files have a custom extension: `*.jsx`.

My guess is that you're doing this because it makes it easier to `require()` files or to use a glob of `*.js` to select all files, maybe in the gulpfile. However, you're using JSX syntax in files that don't have this `*.react.js` extension. My recommendation would be to either make them all have a flat `*.js` or `*.jsx` extension across the board.